README.md

```diff
@@ -1 +1,6 @@
-# cr-contact-form
\ No newline at end of file
+# cr-contact-form
+
+This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
```

Comment:
- [ ] [REQUIRED] Please, it would be better if you add a more descriptive readme following the Professional Repo guidelines. You can check them [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) 🔗 .